kep/42-podgroup-coscheduling/README.md

```diff
@@ -50,8 +50,8 @@ type PodGroupSpec struct {
 	// will not start anyone.
 	MinResources *v1.ResourceList `json:"minResources,omitempty"`
 
-	// MaxScheduleTime defines the maximal time of members/tasks to wait before run the pod group;
-	MaxScheduleTime *metav1.Duration `json:"maxScheduleTime,omitempty"`
+	// MaxScheduleTimeSeconds defines the maximal time of members/tasks to wait before run the pod group;
+	MaxScheduleTime *metav1.Duration `json:"maxScheduleTimeSeconds,omitempty"`
```

Comment:
According to [api-conventions#units](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units), the type should be an integer. Let's make it `*int32`.

And based on:

> We use the fooSeconds convention for durations, as discussed in the units subsection.
> - fooPeriodSeconds is preferred for periodic intervals and other waiting periods (e.g., over fooIntervalSeconds).
> - fooTimeoutSeconds is preferred for inactivity/unresponsiveness deadlines.
> - fooDeadlineSeconds is preferred for activity completion deadlines.

How about renaming it to `ScheduleTimeoutSeconds`?

Reply:
Seem better. Updated

PTAL